bin/estrip

```diff
@@ -410,8 +410,8 @@ while read -r x ; do
 done < <(
 	# Use sort -u to eliminate duplicates for bug #445336.
 	(
-		scanelf -yqRBF '#k%F' -k '.symtab' "$@"
-		find "$@" -type f ! -type l -name '*.a'
+		cut -d ' ' -f1 < "${PORTAGE_BUILDDIR}"/build-info/NEEDED \
+			| sed -e "s:^:${ED}:"
```

Comment:
It looks like it's not going to include `*.a` here, so maybe add that find call back. For reference, I have ncurses-6.2_p20210619 installed, and CONTENTS contains `*.a` files which do not appear in NEEDED.

Reply:
Oh, right! Good spot.